sale_force_invoiced/models/sale_order.py

```diff
@@ -24,3 +24,14 @@ def _compute_invoice_status(self):
             lambda so: so.force_invoiced and so.state in ("sale", "done")
         ).update({"invoice_status": "invoiced"})
         return res
+
+
+class SaleOrderLine(models.Model):
```

Comment:
could you move this code into models/sale_order_line.py dedicated file ?

Reply:
Split in dedicated file done